ruma-events/src/room/power_levels.rs

```diff
@@ -20,6 +20,9 @@ pub type PowerLevelsEvent = StateEvent<PowerLevelsEventContent>;
 #[ruma_event(type = "m.room.power_levels")]
 pub struct PowerLevelsEventContent {
     /// The level required to ban a user.
+    ///
+    /// If you activate the `compat` feature, this field being a string in JSON will give
+    /// you an `Int` here.
```

Comment:
The field is always of type `Int`, maybe this should say sth. like "deserialization will work for stringified integers too".